presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/GroupReference.java

```diff
@@ -46,9 +47,12 @@ public int getGroupId()
     }
 
     @Override
-    public <R, C> R accept(PlanVisitor<R, C> visitor, C context)
+    public <R, C> R accept(IRVisitor<R, C> visitor, C context)
```

Comment:
Why don't just have a second `accept` method. One accepts `IRVisitor`, the other accepts `PlanVisitor`